src/home/diagnostictool/DiagnosticTool.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import * as React from "react";
-import { StyleSheet, View, ScrollView } from "react-native";
+import { StyleSheet, View, ScrollView, TouchableHighlight } from "react-native";
```

Comment:
What is the TouchableHighlight import doing here? Is it being used as a component somewhere on this file? If not then remove it.